include/aca_ovs_l2_programmer.h

```diff
@@ -32,18 +32,18 @@ class ACA_OVS_L2_Programmer {
   int setup_ovs_bridges_if_need();
 
   int create_port(const std::string vpc_id, const std::string port_name,
-                  const std::string virtual_ip, uint tunnel_id, ulong &culminative_time);
+                  const std::string virtual_ip, const std::string virtual_mac,
```

Comment:
Virtual mac address is not a new requirement or information. I was wondering how ACA worked before without passing the mac address, do we have other magic to pass the mac address?

Reply:
VM sends ARP broadcast packet to look for virtual IP -> virtual mac address mapping. Previously, the datapath would multicast this ARP packet to all the VTEPs on the same VPC. We don't want to broadcast/multicast packets in public cloud since controller has already has all the information about virtual IP/virtual mac/hosting IP. With this updated datapath implementation, we finally use this virtual mac for the ARP responder on the local compute node.

Reply:
Cool thanks for the detailed explanation.